zwave_js/CHANGELOG.md

```diff
@@ -1,5 +1,41 @@
 # Changelog
 
+## 0.1.91
+
+### Breaking changes
+
+- Firmware updates may no longer work properly in Home Assistant versions 2023.9.x and earlier due to a breaking change upstream that couldn't be made backwards compatible. They will be fixed in 2023.10.0 .

```

Comment:
```suggestion
- Firmware updates may no longer work properly in Home Assistant versions 2023.9.x and earlier due to a breaking change upstream that couldn't be made backwards compatible. They will be fixed in 2023.10.0.
```

Reply:
I guess we should explain the reason:
Checking for updates no longer requires communication with the device and therefore reduces the risk of corrupting manufacturer information. This also means that updates for battery-powered devices can be detected without waking up the devices.